app/pb_kits/playbook/pb_time/time.rb

```diff
@@ -9,7 +9,7 @@ class Time
 
       prop :time, required: true
       prop :size, type: Playbook::Props::Enum,
-                  values: %w[md sm],
+                  values: %w[lg md sm xs],
```

Comment:
it's my OCD talking here, but can we order these from `xs` to `lg`?

Reply:
You bet.